Documentation: new "faq" section for the namespace offloading logic. (#780)
liqotech/liqo: docs/pages/User/Troubleshooting/Discovery/_index.md

---
title: Discovery 
weight: 4
---

### Which resources will be created during the process?

> Some Kubernetes resources will be created in both the clusters involved in this process.

Reviewer: ```suggestion
Liqo creates the following Kubernetes resources in both the clusters involved in the discovery process.
```

#### In the Home Cluster

| Resource | Name                     | Description |
| -------- | ------------------------ | ----------- |
| Secret   | remote-token-$FOREIGN_CLUSTER_ID | A secret containing a token to authenticate to a remote cluster    |
| Secret   | remote-identity-*        | A secret containing the identity retrieved from the remote cluster |

> > NOTE: these Secret will not be deleted after the ForeignCluster deletion. Do not delete the "remote identit" Secret,

Reviewer: ```suggestion
> NOTE: these Secret will not be deleted after the ForeignCluster deletion. Please do not delete the "remote identity" Secret,
```
